### PR TITLE
Added new enum. Added fields for two classes

### PR DESCRIPTION
**Pull request by zeroambition**

### CODE REVIEW

File: src/main/java/com/scribassu/tracto/domain/ExtramuralLesson.java [hunk 1]
```diff
@@ -6,4 +6,11 @@
 @Data
 @NoArgsConstructor
 public class ExtramuralLesson {
+    private Department department;
+    private String groupNumber;
+    private int day;
+    private int month;
+    private Time timeOfLesson;
```
Comment (Dzeru) [inline src/main/java/com/scribassu/tracto/domain/ExtramuralLesson.java]: rename to lessonTime

File: src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java [hunk 2]
```diff
@@ -6,4 +6,11 @@
 @Data
 @NoArgsConstructor
 public class FullTimeLesson {
+    private Department department;
+    private String groupNumber;
+    private int dayOfWeek;
+    private Time timeOfLesson;
+    private String info;
+    private WeekType typeOfWeek;
```
Comment (Dzeru) [inline src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java]: rename to weekType

File: src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java [hunk 3]
```diff
@@ -6,4 +6,11 @@
 @Data
 @NoArgsConstructor
 public class FullTimeLesson {
+    private Department department;
+    private String groupNumber;
+    private int dayOfWeek;
+    private Time timeOfLesson;
```
Comment (Dzeru) [inline src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java]: rename to lessonTime

File: src/main/java/com/scribassu/tracto/domain/WeekType.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package com.scribassu.tracto.domain;
+
+public enum WeekType {
+    ODD("чис."),
+    EVEN("знам.");
+
```
Comment (Dzeru) [inline src/main/java/com/scribassu/tracto/domain/WeekType.java:7]: add enum for lesson, which take place both in odd and even weeks

Comment (zeroambition) [reply src/main/java/com/scribassu/tracto/domain/WeekType.java:7]: +

File: src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java (hunk 2 above)
Comment (zeroambition) [reply src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java]: +

File: src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java (hunk 3 above)
Comment (zeroambition) [reply src/main/java/com/scribassu/tracto/domain/FullTimeLesson.java]: +

File: src/main/java/com/scribassu/tracto/domain/ExtramuralLesson.java (hunk 1 above)
Comment (zeroambition) [reply src/main/java/com/scribassu/tracto/domain/ExtramuralLesson.java]: +

**Review by Dzeru — APPROVED** [review]

**Pull request merged**